docs/develop/api-reference/index.mdx

```diff
@@ -137,6 +137,29 @@ Sends a ping to the server. This API can be used for checking connectivity to co
 
 </details>
 
+### ItemGetType
+
+For a given key, returns the type (SCALAR, DICTIONARY, LIST, etc.) of the corresponding item, if it exists.
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| cacheName       | String          | Name of the cache.                            |
+| key        | String \| Byte          | Key whose item type should be returned.              |
+
+<details>
+  <summary>Method response object</summary>
+
+  * Cache hit
```

Comment:
These are formatted inconsistently with the dictionary responses. The dictionary ones have 
* Hit
* Miss 
* Error

We should be consistent with the return types. Personally, I prefer hit, miss, error over cache hit, cache miss, and cache error